ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js

```diff
@@ -99,6 +99,12 @@
                     this.widgetObject.setValue(e.target.value);
                     this.setActive();
                 }, this.getWidget());
+                this.widgetObject.addEventListener('input', (e) => {
```

Comment:
I am not sure how is this change related to the current issue.

Reply:
This issue is not directly related to timezone issue, but when we fix the timezone, this issue manifests in one of the test case.
Reason being that we were not clearing the value of the input field when user emptied it. (We do that from calendar icon but I think this was missed when manual entry of date was implemented). 
https://github.com/adobe/aem-core-forms-components/blob/ac6f9b768aa6cd807cbd4b3a40af936aafb7c5d0/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js#L1010